src/types/api/storage.ts

```diff
@@ -228,6 +228,8 @@ export interface TStoragePDisk {
     Whiteboard?: TPDiskStateInfo;
 }
 
+export type Erasure = 'none' | 'block-4-2' | 'mirror-3-dc' | 'mirror-3of4';
```

Comment:
maybe enum and use it in isErasureWithDifferentDC ?

Reply:
I don't like enums, you have to import them everywhere to satisfy TS, while for union string types every option will be suggested by autocomplete